packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -397,13 +397,13 @@ export const formatters: Formatters = {
         event.server_hostname || event.server_addr || event.server_id;
 
       if (!event.interactive) {
-        return `User [${user}] has ended non-interactive session [${event.sid}] on node [${node}] `;
+        return `User [${user}] has ended an non-interactive session [${event.sid}] on node [${node}] `;
```

Comment:
This should be `a non-interactive session` rather than `an non-interactive session`

Reply:
revised: https://github.com/gravitational/webapps/pull/124/commits/dc32916bf82b41ba5dded6dd83f0510a42cec023